source/includes/code-examples/delete-many/DeleteMany.cs

```diff
@@ -26,12 +28,16 @@ public static void Main(string[] args)
         Restore(docs);
     }
 
+    // Deletes all documents that have a Borough value of "Brooklyn"
     private static DeleteResult DeleteMultipleRestaurantsBuilder()
     {
-        // start-delete-many-builders
+        // start-delete-many-builders 
+        // Creates a filter for all documents the have a
+        // Borough value of "Brooklyn"
```

Comment:
I put the CodeWhisper comment below the marker ([against recommendations](https://docs.google.com/document/d/1Bnhh1OsGRUuXo_6yVUASGWiSX6sWuqrEFxPP6sLZDHM/edit#heading=h.xv6glj2lnzjh)) because of the descriptive comment on line 39 - it seemed odd to include one and not the other.